aquarius/events/events_monitor.py

```diff
@@ -349,7 +349,8 @@ def processNewDDO(self, event):
             'value': 0.0,
             'type': '',
             'address': '',
-            'pools': []
+            'pools': [],
+            'isConsumable': 'false'
```

Comment:
should be `''` , not  `false`, `false` is only when there is a pool or exchange but you can't consume because not enough dt